policies/gitlab/repository.rego

```diff
@@ -168,7 +169,6 @@ no_signed_commits {
     is_null(input.push_rules)
 }
 
-
```

Comment:
Related to the code below: since we have both "Project Doesn't Require Code Review" and "Project Doesn't Require Code Review By At Least Two Reviewers" policies, and right now we fire both of them, maybe we should indicate in "Project Doesn't Require Code Review" some text that stresses the fact that 2 reviewers would be preferable and put a link to the two reviewers policy?

Reply:
Sure. How do you link to a policy?